### ISSUE TITLE
Allow disabling hyperparameter tuning

### ISSUE THREAD

**Issue by wielandbrendel** [bethgelab/foolbox#13]
We implemented all attacks to perform some type of internal hyperparameter-tuning. For example, FGSM loops over many step-sizes to find the minimum perturbation. At the same time, some people might want to run the attack with fixed hyperparameters (as has been customary). Should we allow this scenario as well? How difficult/cumbersome would it be to support that?

**Comment by jonasrauber** [comment]
I think this should already be possible by passing an array / list with one entry (the desired epsilon) using the `epsilons` keyword argument when applying the attack. If this works we could still discuss simplifying this / making it more obvious, but it would be quite attack-specific.

**Comment by jonasrauber** [comment]
Our new implementations of the Basic Iterative Method, Projected Gradient Descent and the Momentum Iterative Method all support this (by default they do however optimize the hyperparameters).
